index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+        <title>HTML project</title>
+    </head>
+    <body>
+        <main id="main">
+            <h1 id="title">Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
```

Comment:
Please wrap `h1` and `p`  tags into a `header` tag. This is for semantic reasons and the right position for convention and user experience.